app/src/androidTest/java/com/sdp/movemeet/view/profile/EditProfileActivityTest.java

```diff
@@ -88,6 +90,23 @@ public void updateProfileByEnteringFieldsAndClickingSaveButton() {
 
     }
 
+    @Test
+    public void testAccessFirestoreUsersCollectionForUpdate() {
+
+        try (ActivityScenario<EditProfileActivity> scenario = ActivityScenario.launch(EditProfileActivity.class)) {
```

Comment:
I didn't know we could have a type parameter when launching an `ActivityScenario`, thanks!

Reply:
Yes this is a trick I also used in ProfileActivityTest!